semanticcpg/src/main/scala/io/shiftleft/passes/receiveredges/ReceiverEdgePass.scala

```diff
@@ -31,26 +31,22 @@ class ReceiverEdgePass(graph: ScalaGraph) extends CpgPass(graph) {
     graph.V
       .hasLabel(NodeTypes.CALL)
       .sideEffect { call =>
-        val instanceOption = call
```

Comment:
I like breaking up those chains for better readability. Please revert this part.

Reply:
done